libs/vectorscan/Makefile

```diff
@@ -0,0 +1,82 @@
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=vectorscan
+PKG_VERSION:=5.4.11
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
```

Comment:
local tarballs please.

Reply:
I am confused by this comment.  Is line 12 not OK?  It is present in hyperscan's Makefile: https://github.com/openwrt/packages/blob/master/libs/hyperscan/Makefile#L12

Reply:
Its perfectly fine. :-) You can keep it as it is. Nothing to worry about. There are no statistics, reports that we should switch to something else. If we do, then it needs to be through treewide commit. Codeload was used in the past. It is being used now and most likely it will be used in near future unless someone changes his/her mind and do smth with it.

Reply:
Perfect